<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiscPageUI</class>
 <widget class="QWidget" name="MiscPageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbDoNotShowHeader">
        <property name="styleSheet">
         <string notr="true">QWidget {
  
	color : rgb(0,106,255);
    font:  bold 10pt Calibri;

 }


</string>
        </property>
        <property name="text">
         <string>Ne pas afficher le haut de page</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbDoNotShowFooter">
        <property name="styleSheet">
         <string notr="true">QWidget {
  
	color : rgb(0,106,255);
    font:  bold 10pt Calibri;

 }


</string>
        </property>
        <property name="text">
         <string>Ne pas afficher le bas de page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line1">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: rgb(0,106,255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frameApply">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="pbGlobal">
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font:  10pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
    padding: 1px;
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>Globale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pbPerPage">
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font:  10pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
    padding: 1px;
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>Par page</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="laAllPagesConfig">
        <property name="styleSheet">
         <string notr="true">QWidget {
  
	color : rgb(0,106,255);
    font:  bold 10pt Calibri;

 }


</string>
        </property>
        <property name="text">
         <string>Pour toutes les pages, utiliser
la configuration suivante:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: rgb(0,106,255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QFrame" name="frameReplace">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="pbReplaceLocalWIthGlobal">
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font:  10pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
    padding: 1px;
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>Remplacer la configuration de toutes les pages
avec la configuration globale</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pbReplaceGlobalWithLocal">
        <property name="styleSheet">
         <string notr="true">

QPushButton {
    font:  10pt Calibri;  
    border: 1px solid gray;
    border-radius: 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
    padding: 1px;
 }
 
QPushButton::enabled {
	color : rgb(0,106,255);
    border: 1px solid rgb(0,106,255);
 }

QPushButton:hover:!pressed{

     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QPushButton:pressed{
    background-color:  rgb(220,220,220);

}</string>
        </property>
        <property name="text">
         <string>Remplacer la configuration globale
avec la configuration de cette page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
